src/tabs/Teach__c.tab

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I haven't seen any question that mentions creating a tab of Teach Object. Coul you please verify?